normalize/diff.py

```diff
@@ -183,6 +184,8 @@ def __init__(self, ignore_ws=True, ignore_case=False,
                 Restrict comparison to the fields described by the passed
                 :py:class:`MultiFieldSelector` (or list of FieldSelector
                 lists/objects)
+
+            ``recurse=``\ *BOOL* Do a deeper comparison via recursion
```

Comment:
I'd prefix that statement with "during diff operations, " to make it clear which functionality it affects :)